tests/samplers_tests/test_partial_fixed.py

```diff
@@ -0,0 +1,73 @@
+from typing import Union
+
+import pytest
+
+import optuna
+from optuna.samplers import PartialFixedSampler
+from optuna.samplers import RandomSampler
+from optuna.trial import Trial
+
+
```

Comment:
How about checking that the `ExperimentalWarning` is raised. Please see an example [here](https://github.com/optuna/optuna/blob/master/tests/samplers_tests/tpe_tests/test_sampler.py#L31).

And, you can suppress the `ExperimentaolWarning` in tests like [here](https://github.com/optuna/optuna/blob/master/tests/samplers_tests/tpe_tests/test_sampler.py#L67).